sycl/doc/extensions/README.md

```diff
@@ -1,3 +1,34 @@
 # Extensions
 
-This is where documents can be found that propose extensions to the SYCL specification.
+The directory contains documents that describe DPC++ extensions to SYCL
+specification.
+
+DPC++ extensions status:
+
+|  Extension  |    Status   |   Comment   |
+|-------------|:------------|:------------|
+| [SYCL_INTEL_bitcast](Bitcast/SYCL_INTEL_bitcast.asciidoc) | Proposal | |
+| [C and C++ Standard libraries support](C-CXX-StandardLibrary/C-CXX-StandardLibrary.rst) | Partially supported | |
+| [SYCL_INTEL_data_flow_pipes](DataFlowPipes/data_flow_pipes.asciidoc) | Supported | |
+| [SYCL_INTEL_deduction_guides](deduction_guides/SYCL_INTEL_deduction_guides.asciidoc) | Supported | |
+| [SYCL_INTEL_device_specific_kernel_queries](DeviceSpecificKernelQueries/SYCL_INTEL_device_specific_kernel_queries.asciidoc) | Proposal | |
+| [SYCL_INTEL_enqueue_barrier](EnqueueBarrier/enqueue_barrier.asciidoc) | Proposal | |
+| [SYCL_INTEL_extended_atomics](ExtendedAtomics/SYCL_INTEL_extended_atomics.asciidoc) | Proposal | |
+| [SYCL_INTEL_group_algorithms](GroupAlgorithms/SYCL_INTEL_group_algorithms.asciidoc) | Proposal | |
+| [SYCL_INTEL_group_mask](./GroupMask/SYCL_INTEL_group_mask.asciidoc) | Proposal | |
+| [FPGA selector](IntelFPGA/FPGASelector.md) | Supported | |
+| [FPGA reg](IntelFPGA/FPGAReg.md) | Supported | |
+| [SYCL_INTEL_kernel_restrict_all](KernelRestrictAll/SYCL_INTEL_kernel_restrict_all.asciidoc) | Supported | |
+| [SYCL_INTEL_attribute_style](KernelRHSAttributes/SYCL_INTEL_attribute_style.asciidoc) | Proposal | |
+| [Queue Order Properties](OrderedQueue/OrderedQueue_v2.adoc) | Supported | |
+| [Queue Shortcuts](QueueShortcuts/QueueShortcuts.adoc) | Supported | |
+| [Reductions for ND-Range Parallelism](Reduction/Reduction.md) | Partially supported | |
+| [SYCL_INTEL_relax_standard_layout](RelaxStdLayout/SYCL_INTEL_relax_standard_layout.asciidoc) | Supported | |
+| [SYCL_INTEL_reqd_work_group_size](ReqdWorkGroupSize/SYCL_INTEL_reqd_work_group_size.asciidoc) | Supported | |
+| [SPV_INTEL_function_pointers](SPIRV/SPV_INTEL_function_pointers.asciidoc) | Supported | |
+| [SPV_INTEL_inline_assembly](SPIRV/SPV_INTEL_inline_assembly.asciidoc) | Supported | |
+| [SYCL_INTEL_static_local_memory_query](StaticLocalMemoryQuery/SYCL_INTEL_static_local_memory_query.asciidoc) | Proposal | |
+| [SYCL_INTEL_sub_group_algorithms](SubGroupAlgorithms/SYCL_INTEL_sub_group_algorithms.asciidoc) | Proposal | |
+| [Sub-groups for NDRange Parallelism](SubGroupNDRange/SubGroupNDRange.md) | Proposal | |
```

Comment:
```suggestion
| [Sub-groups for NDRange Parallelism](SubGroupNDRange/SubGroupNDRange.md) | Deprecated | Does not support host device. |
```

Reply:
@Pennycook , @romanovvlad , do we need to add "SubGroups" to the list? https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/SubGroup/SYCL_INTEL_sub_group.asciidoc

Reply:
It's already mentioned, see line 31

Reply:
I think it's not. There are 3 extensions in the directory: Subgroup, SubGroupNDRange and SubgroupAlgorithms and I think we want status per each.

Reply:
@pvchupin is right.  I suggested what the Subgroup extension line could look like here: https://github.com/intel/llvm/pull/1619#issuecomment-621881552

SubGroupNDRange includes some queries and functions that were deprecated when equivalent functionality was introduced in GroupAlgorithms and SubgroupAlgorithms.  We'll eventually drop support for it entirely (and when that happens we can probably remove its row from this table).

Reply:
Applied original comment and added the line John suggested.